docs/api/rate-limits.md

```diff
@@ -51,3 +51,9 @@ The following tables list the rate limits for the NuGet.org API.
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+| API | Limit Type | Limit Value | API usecase | 
```

Comment:
I think `use case` is two words

Reply:
whoops.. right, needs fixing in the rest of the document too.